src/components/SendTokensOne.js

```diff
@@ -208,8 +191,8 @@ class SendTokensOne extends React.Component {
     }
 
     const renderBalance = () => {
-      const balance = hathorLib.wallet.calculateBalance(Object.values(this.props.historyTransactions), this.state.selected.uid);
-      return <span className="ml-3">({t`Balance available: `}{hathorLib.helpers.prettyValue(balance.available)})</span>;
+      const availableBalance = this.state.selected.uid in this.props.tokensBalance ? this.props.tokensBalance[this.state.selected.uid].available : 0;
```

Comment:
As we are in the code, I feel it is way better to use an if-else block instead of the conditional ternary operator. What do you think?

Reply:
Done